components/financial_assistance/app/views/financial_assistance/applicants/other_questions.html.erb

```diff
@@ -1,98 +1,102 @@
-  <div class="col-lg-10 col-md-10 col-sm-10 col-xs-12 row">
+  <div class="col-lg-10 col-md-10 col-sm-10 col-xs-12 row other_questions">
     <div class="col-md-9">
       <p class="alert alert-error hide"></p>
+      
       <div class="row">
-      <div class="col-md-12">
-        <div class="title-icon">
-          <span class="fa-icon fa-stack fa-lg">
-              <i class="fa fa-circle-thin fa-stack-2x ques-icon"></i>
-              <i class="fa fa-question fa-stack-2x "></i>
-          </span>
+        <div class="col-md-12">
+          <div class="title-icon">
+            <span class="fa-icon fa-stack fa-lg">
+                <i class="fa fa-circle-thin fa-stack-2x ques-icon"></i>
+                <i class="fa fa-question fa-stack-2x "></i>
+            </span>
+          </div>
+          <h2 class="fa-darkblue">Other Questions for <%= @applicant.first_name.capitalize %></h2>
         </div>
-        <h2 class="fa-darkblue">Other Questions for <%= @applicant.first_name.capitalize %></h2>
       </div>
-    </div>
+
       <!-- <h2 class="darkblue">Other Questions for <%= @applicant.first_name.titleize %> </h2> -->
       <p>Answer these questions for this person. When you're finished, select CONTINUE.</p>
       <p class="memo">* = required field</p>
 
-        <% if @applicant.present? && @applicant.no_ssn == "1" && @applicant.is_applying_coverage %>
-          <div class="row row-form-wrapper no-buffer row-height fa-align-elements">
-            <div class='col-md-6'>
-              <span class="fa-text-color">Has this person applied for an SSN *</span>
-            </div>
-            <div class='col-md-2'>
-              <%= f.radio_button("is_ssn_applied", true, class: "radio-yml", id: "is_ssn_applied_yes") %>
-              <label class="radio-yml" for="is_ssn_applied_yes">Yes</label>
-            </div>
-            <div class='col-md-2'>
-              <%= f.radio_button("is_ssn_applied", false, class: "radio-yml", id: "is_ssn_applied_no") %>
-              <label class="radio-yml" for="is_ssn_applied_no">No</label>
-            </div>
-          </div>
-
-          <div class="row row-form-wrapper no-buffer row-height fa-align-elements">
-            <div class='col-md-6'>
-              <span class="fa-text-color">Why doesn't this person have an SSN?</span>
-            </div>
-            <div class='col-md-2'>
-              <%= f.text_field :non_ssn_apply_reason, class: "floatlabel", id: "no_ssn_reason", placeholder: "Reason" %>
-            </div>
-          </div>
-        <% end %>
-
+      <% if @applicant.present? && @applicant.no_ssn == "1" && @applicant.is_applying_coverage %>
         <div class="row row-form-wrapper no-buffer row-height fa-align-elements">
           <div class='col-md-6'>
-            <span class="fa-text-color">Is this person pregnant? *</span>
+            <h5 class="fa-text-color">Has this person applied for an SSN *</h5>
```

Comment:
If you're touching a view with hard coded english, you should be putting it in translation strings